packages/ui-components/src/Input/index.ts

```diff
@@ -3,7 +3,6 @@ export * from "./EmailPasswordInput";
 export * from "./CheckboxInput";
 export {
   SingleLineTextInput as TextInput,
```

Comment:
I think there is a linter warning about not _single lining_ this export.

Reply:
Oops, thanks! I'll fix that

Reply:
> `export { SinglineLineTextInput as TextInput, …`
> I think there is a linter warning about not _single lining_ this export.

👏 👏 👏 Excellent pun.

Reply:
lol I didn't even catch that